CMakeLists.txt

```diff
@@ -39,11 +39,7 @@ EXEC_PROGRAM(
 )
 
 IF (SC_GLOBAL_PYTHON_WRAPPER)
-	IF (APPLE)
```

Comment:
Yes it exists, here:
ls -l /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7/Python.h
-rw-r--r--  1 root  wheel  4329 Aug 23  2015 /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7/Python.h

`cat /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7/Python.h`

#ifndef Py_PYTHON_H
#define Py_PYTHON_H